codice/ddf
# DDF-5679 Removed the /login page

#### What does this PR do?
- Removed the /login page
- Changed logout actions to only return one logout url
- Changed OIDC log out to avoid making an ajax call to the OIDC provider

#### Who is reviewing it? 
@bakejeyner @SmithJosh 

#### Select relevant component teams: 
@codice/security 

#### Ask 2 committers to review/merge the PR and tag them here.
@djblue
@garrettfreibott
@stustison

#### How should this be tested?
Needs: https://github.com/codice/ddf-ui/pull/76
- Verify that the old login page is gone
- Verify that logging in with PKI works and basic auth works (shows a username and password pop-up for basic)
- Verify that the error pages don't link to the /login page
- Verify that 403 has a logout link instead that correctly logs you out. The sign back in link should take you back to the page you were attempting to go to.
- Verify logging out with both PKI, Basic, OIDC, and IdP

#### What are the relevant tickets?
Fixes: #5679

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
